content/hegel/reference/existence/development.mdx

```diff
@@ -0,0 +1,221 @@
+its justification therein&mdash;it is at first regarded in its immediacy. In
+regards of this immediacy, existence is _simple_, and therefore the mediation
+that brought it out is put behind it and it appears as "a first". In other
+words, the thought of existence does not need to appeal to `becoming`, or its
+mediation, in order _to be_. To illustrate: to point something out as _existing_
+minimally determines a simple oneness of `being` and `nothing`, but does not, on
+the face of it, point to any `becoming` or movement. There is just existence.
+One simply exists.
+
+### The Unity of Being and Non-being
+
+The simplicity of `existence` is quickly complicated, as the apparent oneness of
+`being` and `nothing` means that more is at work in `existence`, since these
+categories
+[cannot co-exist](/articles/hegel/reference/becoming/development#the-unity-of-being-and-nothing).
+As Hegel goes on to write:
+
+> As it follows upon becoming, existence is in general _being_ with a
+> _non-being_, so that this non-being is taken up into simple unity with being.
+> _Non-being_ thus taken up into being with the result that the concrete whole
+> is in the form of being, of immediacy, constitutes _determinateness_ as such
+> (Hegel 2010, 84/21.97).
+
+Two things are key in this passage. First, `nothing` is recast as `non-being`
+since the category `nothing` strictly _cannot_ be set in contrast with anything,
+much less `being`, as it is _purely_ `nothing`. It thus cannot conceptually
+co-exist with anything else. `Non-being` serves therefore as the part of the
+oneness in which `being` vanishes into and emerges out of `nothing`. As
+`non-being`, `nothing` is involved but only inasmuch as it is encapsulated by
+`being`.
+
+The second matter is that the unity of `being` and `non-being` that constitutes
+`existence` is itself in the form of `being`.
+
+```mermaid
+flowchart TB
+    subgraph being_as_process["<code>being</code>"]
+    being_a["<code>being</code>"] <---> nothing_a["<code>non-being</code>"]
+    end
+```
+
+The fact that `being` and `non-being` are in the form of `being` ensures their
+unity while also containing their difference. This unity clearly is concrete
+since it both brings together as well as holds apart divergent elements. This is
+exactly what establishes _determinateness_ for Hegel. But how does it achieve
+that? What exactly is implied in this immediate form of determinateness?
+
+### Determinateness as a Concrete Whole
+
+> The _whole_ is likewise in the form or _determinateness_ of being, since in
+> becoming being has likewise shown itself to be only a moment – something
+> sublated, negatively determined. It is such, however, _for us_, _in our
+> reflection_; not yet as _posited_ in it. What is posited, however, is the
+> determinateness as such of existence, as is also expressed by the _da_ (or
+> “there”) of the _Dasein_ (Hegel 2010, 84/21.97).
+
+This passage is tricky since since it appears that Hegel looks ahead from the
```

Comment:
it seems to me that Hegel has to look ahead because the matter at hand (existence as the whole, by the whole, the logic understands the determinateness of being) projects what it will come next. Hegel here warns us that